java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -64,7 +64,7 @@
 
     @Override
     public void run() {
-        log.debug("Engineer started warrant {}", _warrant.getDisplayName());
+        if (debug) log.debug("Engineer started warrant {}", _warrant.getDisplayName());
```

Comment:
That was just an attempt to make runtime a bit more efficient.  i guess I am mistaken.  I'll submit another PR to change this and a few others.